src/pattern-library/components/patterns/data/Scroll.js

```diff
@@ -0,0 +1,187 @@
+import { Scroll, ScrollContainer, ScrollContent } from '../../../../next';
+import Library from '../../Library';
+import Next from '../../LibraryNext';
+
+import { SampleListElements } from '../samples';
+
+export default function ScrollPage() {
+  return (
+    <Library.Page
+      title="Scroll components"
```

Comment:
The `<title>` for this page reads "Scroll" after the entry in the navigation list rather than "Scroll components".

Reply:
Changed to `Scroll`